### PR TITLE
Add fallback formatting

### PR DESCRIPTION
**Pull request by sebwas**
This pull request aims to add [formatting](https://kazupon.github.io/vue-i18n/guide/formatting.html) of fallback messages if that's desired (it's completely controllable by the i18n options and the default is to turn it off (so that it's 100% backwards compatible))

Turn it on by setting the `formatFallbackMessages` to `true`.

The use case is simple: When using the whole string to be translated as key, you will want to format the messages nonetheless:
```html
<div>
  {{ $t('Hello {name}!', { name: 'world' }) }}
</div>
```

### CODE REVIEW

**Comment by sebwas** [comment]
Any comments, @kazupon?

**Comment by kazupon** [comment]
@sebwas 
Sorry, late reply. 🙇 

That's a very interesting feature request!
If the specified key cannot be found, it's convenient to interpolate.

We can use that feature for key like message (e.g. `This is {name}`). We cannot use for key like path. We need to notice to use this feature.
Some documentation updates are required to provide this feature.
Can you update docs? 🙏

File: src/index.js [hunk 1]
```diff
@@ -274,6 +278,13 @@ export default class VueI18n {
         )
       }
     }
+
+    if (this._formatFallbackMessages) {
+      const parsedArgs = parseArgs(...values)
+
+      return this._render(key, 'string', parsedArgs.params, key)
+    }
+
```
Comment (kazupon) [inline src/index.js]: https://github.com/kazupon/vue-i18n/pull/637/files#diff-1fdf421c05c1140f6d71444ea2b27638R266
We need to change from `_warnDefault `, due to have some behavior.

Comment (sebwas) [reply src/index.js]: I don't exactly understand what you mean. Do you mean we should change the wording here, or?

**Comment by AlexandreBonaventure** [comment]
Hello what about this issue #296 ? I think this would be worth to reopen, can I submit a PR ?

**Pull request merged**